repository.json

```diff
@@ -623,7 +623,34 @@
 					"tags": true
 				}
 			]
+		},
+		{
+			"details": "https://github.com/sublimelsp/LSP-leo",
+			"name": "LSP-leo",
+			"author": "aleohq",
+			"homepage": "http://leo-lang.org/",
+			"readme": "https://raw.githubusercontent.com/sublimelsp/LSP-leo/master/README.md",
+			"issues": "https://github.com/AleoHQ/leo-plugins/issues",
+			"description": "A leo language features extension.",
+			"labels": [
+				"aleo",
+				"leo",
+				"auto-complete",
+				"linting",
+				"code navigation",
+				"intellisense",
+				"color scheme",
+				"language syntax",
+				"snippets",
+				"theme"
+			],
+			"releases": [
+				{
+					"sublime_text": ">=4070",
+					"tags": true
+				}
+			]
```

Comment:
I'm not sure why this json object isn't sitting between the other json objects in the json array. Maybe it works? I don't know. Just put it between the others (in alphabetical order).